tests/basictests.cpp

```diff
@@ -892,8 +1079,7 @@ namespace dom_api {
     if (doc["a"].as_uint64_t().first != 1) { cerr << "Expected uint64_t(doc[\"a\"]) to be 1, was " << doc["a"].first << endl; return false; }
 
     UNUSED document::element val;
-    // tie(val, error) = doc["d"]; fails with "no viable overloaded '='" on Apple clang version 11.0.0
-    doc["d"].tie(val, error);
+    tie(val, error) = doc["d"];
```

Comment:
See the CI tests: this syntax fails under libc++...

Reply:
Annoyingly, I might add...

Reply:
Yeah, I did some copy/pasting of a few tests when I moved it around, I'll restore what you have in there.